### ISSUE TITLE
Support for reCAPTCHA

### ISSUE THREAD

**Issue by jimpdx** [nciske/salesforce-wordpress-to-lead#39]
We just started getting a flood of new leads from our site that are SPAM bots. Is there a plan to support a simple "I am not a robot" or standard-style RECAPTCHA?

**Comment by jimpdx** [comment]
I should clarify - I am referring to the "NO CAPTCHA reCAPTCHA" from Google as opposed to the image detection style...

**Comment by jimpdx** [comment]
https://www.google.com/recaptcha/intro/index.html

**Comment by nciske** [comment]
Yes. That's definitely in the plans. May be in the next release (or shortly after) as its going to be a major release and I know the captcha is a pain point for many. 

Release should be in the next week or two.
